recipes/tllist/all/conanfile.py

```diff
@@ -0,0 +1,58 @@
+import os
+
+from conans import ConanFile, Meson, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.44.0"
+
+
+class TllistConan(ConanFile):
+    name = "tllist"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://codeberg.org/dnkl/tllist"
+    description = "A C header file only implementation of a typed linked list."
+    topics = ("list", "utils", "tllist")
+    generators = ("pkg_config")
+    settings = "os", "arch", "build_type"
+    no_copy_source = True
+    _meson = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def validate(self):
+        if self.settings.os != "Linux":
+            raise ConanInvalidConfiguration("Only Linux supported")
+
+    def build_requirements(self):
+        self.tool_requires("pkgconf/[>=1.7.4]")
+        self.tool_requires("meson/[>=0.60.0]")
+        self.tool_requires("ninja/[>=1.10.0]")
```

Comment:
We don´t use version ranges for requirements... but for pure build-requires it is something we could consider if we start to use always two profiles (not-yet). wdyt @danimtb ?

Reply:
> Hi! This is failing at the `conan export "recipes/tllist/all/conanfile.py" tllist/1.0.5@` command. Let me try it locally.

Thank you for the quick response !